lib/api/request/login_request.dart

```diff
@@ -2,7 +2,7 @@ import 'package:json_annotation/json_annotation.dart';
 
 part 'login_request.g.dart';
 
-const String grantType = "password";
+const String grantTypePassword = "password";
```

Comment:
(Optional) How about moving this to `AuthenticationRepository` and making it private since all its usages are there?